plugins/GSdx/GSRenderer.cpp

```diff
@@ -221,22 +221,27 @@ bool GSRenderer::Merge(int field)
 		src[i] = GSVector4(r) * scale / GSVector4(tex[i]->GetSize()).xyxy();
 		src_hw[i] = (GSVector4(r) + GSVector4 (0, y_offset[i], 0, y_offset[i])) * scale / GSVector4(tex[i]->GetSize()).xyxy();
 
-		GSVector2 off(0, 0);
+		GSVector2 offset(0, 0);
+		GSVector2i diff(dr[i].left - baseline.x, dr[i].top - baseline.y);
 
 		// Time Crisis 2/3 uses two side by side images when in split screen mode.
-		off.x = tex[i]->GetScale().x * (dr[i].left - baseline.x);
+		// Though ignore cases where baseline and display rectangle offsets only differ by 1 pixel, causes blurring and wrong resolution output on FFXII
```

Comment:
I'm not sure it's "wrong" output, people just don't like blurring (it probably looked ok on TV). It's interesting that FFXII implements that technique horizontally instead of vertically like most other games.